recipes/chemps2/build.sh

```diff
@@ -0,0 +1,71 @@
+# set -x
+
+LAPACK_INTERJECT="${PREFIX}/lib/libmkl_rt.so"
+# link against conda MKL & GCC
+# if [ "$blas_impl" = "mkl" ]; then
+#     LAPACK_INTERJECT="${PREFIX}/lib/libmkl_rt.so"
+# else
+#     LAPACK_INTERJECT="${PREFIX}/lib/libopenblas.so"
+# fi
+# ALLOPTS="-gnu-prefix=${HOST}- ${OPTS}"
+
+# configure
+${PREFIX}/bin/cmake \
+    -H${SRC_DIR} \
+    -Bbuild \
+    -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+    # -DCMAKE_BUILD_TYPE=Release \
+    # -DCMAKE_C_COMPILER=icc \
```

Comment:
fyi, cmake can't tolerate commented lines interspersed with active lines. need to move them all aside and always keep the last active one w/o a line continuation.